components/inputs/input-label-styles.js

```diff
@@ -8,8 +8,8 @@ export const inputLabelStyles = css`
 		font-weight: 700;
 		letter-spacing: 0.2px;
 		line-height: 1rem;
-		margin: 0 0 7px 0;
-		padding: 0;
+		margin: 0;
+		padding: 0 0 7px 0;
```

Comment:
Why was this change necessary? A potentially unintended consequence of using padding instead of margins is that it'll no longer "collapse" with the element below the input.

Reply:
This was needed to fix an issue where hovering in the middle between the label and the input would not enable the hover styles. 

Though it may not be needed if we add the changes proposed in [US148080](https://rally1.rallydev.com/#/?detail=/userstory/684245679355&fdp=true) to enable hover styles when hovering anywhere in the input-text component. So we could also leave it as-is for now.

Reply:
Cool -- yeah as long as it's not too janky as-is, that sounds like a good plan!